misc/tests/test-lp-config.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
This is copied from test-standard-configs.py. Maybe one can outsource some functions into another file. What do you want?

Reply:
I suggest to reuse test-standard-configs.py. In it, we can check for an environment variable `DOWNWARD_TEST_LP` and if it's set, we add the LP configs to the list of configs. Then we set the environment variable in the lp toxenv.

Reply:
Good idea. Let's wait if the action succeeds.

Currently I check whether the variable is set (not the value of the variable), I am not certain whether this is good style. Should I enforce that the user has a certain value in the variable (e.g. true)

Reply:
I think checking for existence is quite common for env vars.